### PR TITLE
Editorial: Remove ambiguous statement from Date.UTC

### PR DESCRIPTION
**Pull request by leobalter**
The word optionally might be taken as implementation defined to use
the respective arguments it refers to. Removing it makes it clear
the following steps will be taken when Date.UTC is called with 3 or
more arguments.

cc @jugglinmike

### CODE REVIEW

File: spec.html [hunk 1]
```diff
@@ -25647,7 +25647,7 @@
       <!-- es6num="20.3.3.4" -->
       <emu-clause id="sec-date.utc">
         <h1>Date.UTC ( _year_, _month_ [ , _date_ [ , _hours_ [ , _minutes_ [ , _seconds_ [ , _ms_ ] ] ] ] ] )</h1>
-        <p>When the `UTC` function is called with fewer than two arguments, the behaviour is implementation-dependent. When the `UTC` function is called with two to seven arguments, it computes the date from _year_, _month_ and (optionally) _date_, _hours_, _minutes_, _seconds_ and _ms_. The following steps are taken:</p>
+        <p>When the `UTC` function is called with fewer than two arguments, the behaviour is implementation-dependent. When the `UTC` function is called with two to seven arguments, the following steps are taken:</p>
```
Comment (ljharb) [inline spec.html]: also i think we switched to US spellings - ie "behavior" instead of "behaviour" :-)

Comment (claudepache) [inline spec.html]: > i think we switched to US spellings

I think we use OED spelling (whose main difference with British spelling is verbs ending in _-ize_). See:
https://bugs.ecmascript.org/show_bug.cgi?id=2467
https://en.wikipedia.org/wiki/Oxford_spelling#Usage

Comment (allenwb) [inline spec.html]: > I think we use OED spelling (whose main difference with British spelling is verbs ending in -ize). See:

Exactly.  At least that is the way it unless something changed over the last year.  ISO says standards should be written use OED English spellings.

**Comment by leobalter** [comment]
@ljharb I couldn't live with inconsistency so after fixing the "behaviour" on my change, I did another commit replacing other uses of behaviour. That includes a link that will move from `sec-ordinary-and-exotic-objects-behaviours` to `sec-ordinary-and-exotic-objects-behaviors`.

@bterlson the extra commit is just a plus and please ignore and discard it if it blocks the main proposed change.

**Comment by leobalter** [comment]
@ljharb I couldn't live with inconsistency so after fixing the "behaviour" on my change, I did another commit replacing other uses of behaviour. That includes a link that will move from `sec-ordinary-and-exotic-objects-behaviours` to `sec-ordinary-and-exotic-objects-behaviors`.

@bterlson the extra commit is just a plus and please ignore and discard it if it blocks the main proposed change.

On the OED spelling: I can't find any open access for it. Does Ecma has any legal form to browse it or should I subscribe it? Without consulting the OED these definitions will remain cloudy.

**Comment by leobalter** [comment]
Ok, I've got word from someone with access to an OED from the 70s and _behaviour_ is the correct option.

I reverted the "behavior" commits only to avoid force push, please discard before merging.

File: spec.html (hunk 1 above)
Comment (ljharb) [inline spec.html]: Ah, thanks for clarifying. I wasn't familiar with the distinction :-)

Comment (ljharb) [inline spec.html]: (That means we have some "behavior" instances to clean up tho)

**Comment by ljharb** [comment]
There's no need to ever avoid force pushing - probably easiest to just rebase them out.

**Comment by bterlson** [comment]
Just pushed a fix for the improper usage of "behavior".

**Pull request merged**